Added initial EVSE device type to matter-devices.xml. Updated energy-…

jamesharrow opened the pull request:
The EVSE Device type had been missing, and the energy-management-app ZAP file was therefore generating errors.
Other clusters on Root node were missing and have been added to stop the warnings ZAP is now generating.

cecille commented on examples/energy-management-app/energy-management-common/energy-management-app.zap:
```diff
@@ -746,6 +746,266 @@
           ]
         },
+        {
+          "name": "Localization Configuration",
```
were these added manually, or was this a zap addition? ie, do you WANT these clusters, or is this a weird zap side effect?

jamesharrow commented:
I added it manually because ZAP now spits out lots of Red siren 🚨 symbols if the root node is missing these.In one case the esp32 build wouldn’t build for me until I fixed this.On 30 Jan 2024, at 18:30, C Freeman ***@***.***> wrote:﻿
@cecille approved this pull request.



In examples/energy-management-app/energy-management-common/energy-management-app.zap:
> @@ -746,6 +746,266 @@
             }
           ]
         },
+        {
+          "name": "Localization Configuration",

were these added manually, or was this a zap addition? ie, do you WANT these clusters, or is this a weird zap side effect?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
